spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/RefBuilder.scala

```diff
@@ -0,0 +1,58 @@
+package com.datastax.spark.connector.util
+
+import com.datastax.spark.connector.cql.{AuthConfFactory, CassandraConnectionFactory}
+
+case class ConfigParameter[T](
```

Comment:
nit: Since this is a file devoted to reference builder and `ConfigParameter` is a more general class, I would extract it to a separate file.